src/Writers/HTMLWriter.jl

```diff
@@ -598,17 +598,17 @@ the page also happens to be a level 1 heading. In that case it is assumed to be
 the page title and is dropped from the list of subsections.
 """
 function collect_subsections(page::Documents.Page)
```

Comment:
Actually, the docstring should probably be updated, to make it clear that it's the first level 1 heading that gets dropped.